controllers/index.go

```diff
@@ -23,15 +24,23 @@ func NewIndexController(s *myradio.Session, c *structs.Config) *IndexController
 // Get handles the HTTP GET request r for the index page, writing to w.
 func (ic *IndexController) Get(w http.ResponseWriter, r *http.Request) {
 	// This is where any form params would be parsed
-
 	model := models.NewIndexModel(ic.session)
 
-	data, err := model.Get()
+	currentAndNext, banners, err := model.Get()
```

Comment:
Not sure what to do here :) @Brookke You able to make the changes here so we can merge?